openapi/src/json-schemas/models/definitions/api-version.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "x-api-version",
+    "type": "string",
+    "enum": ["1.0.0", "2.0.0"]
```

Comment:
Lets go with the date based versioning. To kick things off we can start with:

`"enum": ["2023-05-17"]`